timemachine/ff/handlers/nonbonded.py

```diff
@@ -67,14 +68,14 @@ def oe_generate_conformations(oemol):
     omega.SetIncludeInput(False)
     # needed to preserve the atom ordering
     omega.SetCanonOrder(False)
-    omega.SetSampleHydrogens(True)
+    omega.SetSampleHydrogens(sample_hydrogens)
     omega.SetEnergyWindow(15.0)
     omega.SetMaxConfs(800)
     omega.SetRMSThreshold(1.0)
 
     has_confs = omega(oemol)
     if not has_confs:
-        raise Exception(f"Unable to generate conformations for charge assignment {oemol.GetTitle()}")
+        raise Exception(f"Unable to generate conformations for charge assignment for {oemol.GetTitle()}")
```

Comment:
super nit: Might be good to wrap `oemol.GetTitle()` in single quotes in case there is no title for the mol

Same for down on ln109